solr/distribution/build.gradle

```diff
@@ -92,6 +82,14 @@ task computeChecksums(type: Checksum) {
   outputDir = file("${buildDir}/checksums")
 }
 
+task signReleaseArchives(type: Sign) {
+  sign distTarTask
+  sign distZipTask
+
+  // The source tgz is not an archive task so be explicit about the outputs to sign.
+  dependsOn tasks.assembleSourceTgz
+  sign tasks.assembleSourceTgz.destination
```

Comment:
I in-sourced the one task from external file into this to simplify...

Reply:
Is it weird that source tgz is assembled here, while binary tgzs are assembled in `packaging`? Could easily move this over I guess.

Reply:
I think its fine. Packaging can mainly just be considered the binary release module